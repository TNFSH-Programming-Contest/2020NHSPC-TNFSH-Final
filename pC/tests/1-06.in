5 10 4
4 1 376204
1 2 123561
2 3 55024
3 5 711515
1 3 454967
1 5 654789
4 2 154255
3 4 603537
2 5 645359
5 4 898795
