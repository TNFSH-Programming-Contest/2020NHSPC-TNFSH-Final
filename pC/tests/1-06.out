376204 154255 418558 0 898795 